src/components/NavBarDropdown/formatDropDownMenuItems.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { MenuProps } from "antd";
+import classNames from "classnames";
+
+import { NavBarAnchorType, NavBarDropdownItem, NavBarDropdownItemGroup } from "../../component-queries/types";
+import LinkOut from "../LinkOut";
+import PdfAnchor from "../PdfAnchor";
+import { COLLECTION_COMPONENTS_MAP } from "./constants";
+
+const {
+    hasBottomBorder,
+    groupTitle
+} = require("../../style/navbarDropdown.module.css");
+
+const isGroupedItems = (
+    items: NavBarDropdownItem[] | NavBarDropdownItemGroup[]
+): items is NavBarDropdownItemGroup[] => {
+    return items.length > 0 && "options" in items[0];
+};
+
+const getItemComponent = (item: NavBarDropdownItem) => {
+    if (item.anchorType === NavBarAnchorType.Download) {
+        const filename = item.href.split('/').pop();
+        return <PdfAnchor label={item.label} href={`/pdf/${filename}`} />;
+    }
+
+    if (item.anchorType === NavBarAnchorType.Internal) {
+        return <LinkOut label={item.label} href={item.href}/>;
```

Comment:
this is where link out looks confusing, i didn't notice the `newTab=true` below so it seemed weird to have Internal => LinkOut